.github/workflows/helm-chart-ci-ignore.yaml

```
@@ -0,0 +1,61 @@
+name: Helm Chart CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [synchronize, opened, reopened, edited]
```

```suggestion
    types: [synchronize, opened, reopened]
```

Edited runs when the description or title changes.